codex-ui/src/vue/components/alert/Alert.types.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Various alert type
+ */
+export type Alertype = 'success' | 'error' | 'warning' | 'info' | 'default';
+
+/**
+ * alert configuration
+ */
+export interface AlertOptions {
+
+  id?: number;
+  /**
+   * Custom icon class to be used.
+   *
+   */
+  icon?: string;
+  /**
+   * content to be rendered
+   */
+  message?: string;
+  /**
+   *  Type of the alert.
+   *
+   *  Can be any of `(success, error, default, info, warning)`
+   */
+  type?: Alertype;
```

Comment:
as i can see, this field is unused

Reply:
it's used but passed as props, hence it's passed as a constant  in usealert return values